web-frontend/src/components/Dependencies.vue

```diff
@@ -155,9 +155,9 @@ export default {
   methods: {
     //
     clean_changelog(changelog) {
-      let res = changelog.replaceAll(/(#)*/g, "");
+      let res = changelog.replace(/(#)*/g, "");
       // strip markdown links
-      res = res.replaceAll(/\[([^\]]+)\]\([^)]+\)/g, "$1");
```

Comment:
note: you might not see how this affect the frontend if you're not running the backend with a github token. 

background: we show github changelog (if one is available) in two places: 

1. in the "changelog" column, when you hover one it'll show a popover with the content.
2. in the review tab of a dependency (when you try to update).
 
These changelogs are usually formatted using markdown, which is fine if you show the text in the body, but it doesn't display nice in a popover. So my hack was about quickly removing the markdown stuff (which work well enough at that point).

your change: remove this feature, you now only replace the first occurence of # (for titles)) and links (`[text](link)`) instead of all occurences.

do we care anyway? I suggest that we get rid of the changelog column, as we already display that information when we want to review a dep.

Reply:
IMO, the popover is not really useful, plus we have two different things currently it seems:
![image](https://user-images.githubusercontent.com/10077203/114367971-41145180-9b7d-11eb-9ee3-45bfb2f2e54b.png)

both appear when hovering over the "preview" link, which does nothing when clicked.
Notice how the first one is cut after a few words?
And notice how the second one is super big (it actually goes off screen, so it's not super useful)

IMO that should change: when clicked it should display the full changelog, when hovered I'm not sure what's best, wdyt?

Reply:
>  I suggest that we get rid of the changelog column, as we already display that information when we want to review a dep.

Reply:
Also, discussing with @AnomalRoil 
I am running an OLD version of chromium that doesn't support replaceAll (see: https://caniuse.com/?search=replaceAll). 
so it looks like, I can revert these specific changes and focus on the primary issue that triggered this PR in the first place (date + commit info not showing).

Reply:
I think there is some conversation we can have regarding the desired experience for interacting with changelogs, but let's deal with that in a separate issue/pull-request, if you all don't mind.

I'll resolve this particular conversation (but feel free to 're-open' it if you think it needs to be addressed in this PR).

Reply:
Also gotta keep in mind that changelogs might not reflect reality... so we can’t put too much emphasis there